src/main/features/claim/routes/pay.ts

```diff
@@ -79,7 +81,13 @@ async function successHandler (res, next) {
   }
 
   if (!claimIsAlreadyFullyPersisted) {
-    await claimStoreClient.saveClaim(draft, user)
+    const roles: string[] = await claimStoreClient.retrieveUserRoles(user)
+    const admissionsAllowed: boolean = await featureTogglesClient.isAdmissionsAllowed(user, roles)
```

Comment:
No need to call `featureTogglesClient.isAdmissionsAllowed` if `claimStoreClient.retrieveUserRoles(user)` evaluates  into empty list of roles. Please wrap `featureTogglesClient.isAdmissionsAllowed(user, roles)` with `if` statement.